### DISCUSSION TITLE
What's the status of the 6.0 GA release?

### DISCUSSION THREAD

**Discussion by David-Engel** [dotnet/SqlClient#3082] [General]
In an effort to be transparent, I thought I would drop a discussion note on where we are with the 6.0 GA release.

The original goal had been to release it mid to late November. That was delayed getting PRs merged and then inserting a preview release for validation, pushing GA to early December.

Holidays are never a good time to release, so things were getting tight.

While trying to get the final MDS.SNI 6.0 GA package out, we ran into issues in those pipelines due to new/stricter compliance requirements and an issue in the latest Windows SDK update for arm32, which all took several days to figure out. At this point, we finally just had to move the date to January due to vacations and holidays.

The date on the [milestone](https://github.com/dotnet/SqlClient/milestones) has been updated.

**Comment by MichelZ** [comment]
As 6.0 has been branched away, maybe some of the 7.x items can be merged now?

**Reply by David-Engel** [reply]
That was the goal of branching for 6.0. yes. Have some patience for the team, but feel free to re-request a review from the @dotnet/sqlclientdevteam if you feel a PR has been overlooked.

**Reply by TonyValenti** [reply]
Thanks for the update. 
I am really interested in the update that fixes performance for reading large blobs from the database. I saw comments and a PR for it related to N! performance.

Will that be part of 6.0 or 7.0?

**Reply by MichelZ** [reply]
7.0, see the milestone on the PR

**Comment by Tornhoof** [comment]
It's now on nuget: https://www.nuget.org/packages/Microsoft.Data.SqlClient

For some reason it does not show up here:
https://github.com/dotnet/SqlClient/releases

**Reply by ErikEJ** [reply]
@Tornhoof Look in the release/6.0 branch for release notes

**Reply by Tornhoof** [reply]
@ErikEJ Thanks, yeah I've seen the release notes. I was just surprised that it does not show up on the appropriate github page.

**Reply by MikeFarrington** [reply]
What happened to the 6.0.0 release? I had downloaded a 6.0.0 NuGet package on January 15th (but had to rollback because SMO didn't support it yet).  Now I don't even see it anymore on NuGet.org, and its release page here has gone missing?

**Reply by MichelZ** [reply]
https://github.com/dotnet/SqlClient/issues/3121#issuecomment-2598831458

> Cheena has a PR open for the changes. We found an issue with the 6.0.0 release right after it was published (transitive copying of the SNI DLLs was broken) so there is a scramble to fix it and publish 6.0.1 ASAP. The docs will get updated after 6.0.1 goes out. I just unlisted 6.0.0.

**Comment by minoseah629** [comment]
please update documentation to provide the difference between the preview versions and the stable release released yesterday to nuget.org and what's within 6.0.1 released today

![image](https://github.com/user-attachments/assets/fe030e06-ace7-4637-ba82-6fcce6a5a209)

**Reply by ErikEJ** [reply]
@minoseah629 Have a look at the PR in progress

**Comment by paulmedynski** [comment]
SqlClient 6.0.0 was released with a broken dependency on its associated SNI package, so it was removed from NuGet and the documentation, and replaced with [6.0.1](https://github.com/dotnet/SqlClient/releases/tag/v6.0.1).  The subsequent [6.0.2 release](https://github.com/dotnet/SqlClient/releases/tag/v6.0.2) has also been documented and published.
